stubs/atheris/atheris/utils.pyi

```diff
@@ -0,0 +1,13 @@
+from typing import IO
+
+def path() -> str: ...
+
+class ProgressRenderer:
+    def __init__(self, stream: IO[str], total_count: int) -> None: ...
```

Comment:
`IO` and other concrete classes should be avoided in argument annotations. This protocol should work instead:

```python
@type_check_only
class _Writer(Protocol):
    def isatty(self) -> bool: ...
    def write(self, content: bytes, /) -> object: ...
    def flush(self) -> object: ...
```